cdap-docs/application-templates/source/etl/templates/transformations/structuredrecordtogenericrecord.rst

```diff
@@ -6,6 +6,28 @@
 Transformations: StructuredRecordToGenericRecord 
 =================================================
 
-.. rubric:: Description: Transforms a StructuredRecord into an Avro GenericRecord
+.. rubric:: Description
 
-As the description says, it turns a StructuredRecord into an Avro GenericRecord.
+Transforms a StructuredRecord into an Avro GenericRecord. StructuredRecord is
+the Java class that all built-in plugins work with. Most StructuredRecords can be
+directly converted to a GenericRecord. The exception is if the StructuredRecord contains
```

Comment:
An exception is a StructuredRecord that contains a map field with keys that are not of type 'string'.